tests/override_class.phpt

```diff
@@ -5,7 +5,7 @@ zip
 --INI--
 shadow.override=ziparchive::open@w0,ziparchive::addfile@0
 --SKIPIF--
-<?php if (!extension_loaded("shadow")) print "skip"; ?>
+<?php if (!extension_loaded("shadow") && !extension_loaded("zip")) print "skip"; ?>
```

Comment:
I see that all tests in this suite check if `extension_loaded('shadow')`, but what's the point? If we can't load the extension being tested and skip all tests, and mark the suite as passed, does it mean we've tested everything we could? I think it makes sense (not as part of this issue) try removing this check and seeing if we can avoid it.